config/mcp_server.yml

```
@@ -0,0 +1,157 @@
+# ClickUp MCP Server Configuration for Project Archangel
+# Enables AI-enhanced task management through Model Context Protocol
+
+server:
+  # Server connection settings
+  host: "127.0.0.1"
+  port: 3231
+  endpoint: "/mcp"
+  transport: "http"  # Options: http, sse
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)